content/en/docs/concepts/storage/volumes.md

```diff
@@ -1247,6 +1247,13 @@ in `containers[*].volumeMounts`. Its values are:
   (unmounted) by the containers on termination.
   {{< /warning >}}
 
+  {{< warning >}}
+  Mount propagation is a low-level feature that does not work consistently on all
+  volume types. We recommend using it only with `hostPath` or in-memory `emptyDir`
```

Comment:
actually, is it specifically in-memory emptydir, or any emptydir type?

Reply:
I've used it with in-memory emptydir. I think it should work with any type, but do others have an experience?

Reply:
Nit: [avoid using “we“](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we)

Reply:
Done